migrations/00004_notifications/index.sql

```diff
@@ -0,0 +1,82 @@
+DROP TABLE IF EXISTS notification_types,
+    notification_subscriptions,
+    notification_mediums,
+    notification_medium_configurations CASCADE;
+
+--------------------------------------------
+-- Notification types, e.g.INCOMING_TOKEN --
+--------------------------------------------
+CREATE TABLE notification_types(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
+);
+
+-- TODO: Confirm these types
+INSERT INTO notification_types (name) VALUES
+    ('DELETED_MULTISIG_TRANSACTION'),
+    ('EXECUTED_MULTISIG_TRANSACTION'),
+    ('INCOMING_ETHER'),
+    ('INCOMING_TOKEN'),
+    ('MESSAGE_CREATED'),
+    ('MODULE_TRANSACTION'),
+    ('NEW_CONFIRMATION'),
+    ('MESSAGE_CONFIRMATION'),
+    ('OUTGOING_ETHER'),
+    ('OUTGOING_TOKEN'),
+    ('PENDING_MULTISIG_TRANSACTION'),
+    ('SAFE_CREATED');
```

Comment:
These are currently mapped 1:1 with out hooks. We need to determine which we want to keep and/or modify. For example, the Transaction Service [dispatches a custom `CONFIRMATION_REQUIRED`](https://github.com/safe-global/safe-transaction-service/blob/c6b20eea6a5ffa14b2cc6c9879e69cca7e7e0d11/safe_transaction_service/notifications/tasks.py#L189) as well.